docs/content/integration/openid-connect/gitea/index.md

```diff
@@ -89,10 +89,12 @@ To configure [Gitea] to utilize Authelia as an [OpenID Connect 1.0] Provider:
 
 {{< figure src="gitea.png" alt="Gitea" width="300" >}}
 
-Alternatively, you can add the authentication source via command line (from the directory of the `gitea` binary):
+#### CLI
+
```

Comment:
_**Important Note:** Please refer to the [Gitea CLI Guide](https://docs.gitea.com/administration/command-line) regarding the correct usage of the CLI._